turborepo-tests/integration/tests/remote-config.t

```diff
@@ -0,0 +1,42 @@
+Setup
+  $ . ${TESTDIR}/../../helpers/setup.sh
+  $ . ${TESTDIR}/_helpers/setup_monorepo.sh $(pwd)
+
+Run test run
+  $ ${TURBO} run build --__test-run | jq .remote_config
+  {
+    "token": null,
+    "team_id": null,
+    "team_slug": null,
+    "api_url": "https://vercel.com/api",
+    "use_preflight": false,
+    "timeout": 20
+  }
+
+Run test run with api overloaded
+  $ ${TURBO} run build --__test-run --api http://localhost:8000 | jq .remote_config.api
+  null
```

Comment:
wait why is this null? shouldn't it be localhost:8000?

Reply:
Ah good catch, had the wrong field (`api` instead of `api_url`)